qa/suites/rgw/multisite-upgrade/reef-x/overrides.yaml

```diff
@@ -0,0 +1,23 @@
+overrides:
+  ceph:
+    log-ignorelist:
+    - \(PG_AVAILABILITY\)
+    wait-for-scrub: false
+    conf:
+      client:
+        setuser: ceph
+        setgroup: ceph
+        debug rgw: 20
+        rgw crypt s3 kms backend: testing
+        rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+        rgw crypt require ssl: false
+        rgw sync log trim interval: 0
+        rgw curl low speed time: 300
+        rgw md log max shards: 4
+        rgw data log num shards: 4
+        rgw sync obj etag verify: true
+  rgw:
+    compression type: random
+  rgw-multisite-tests:
+    branch: reef # run reef branch of tests
```

Comment:
would you be willing to pull my commit from https://github.com/ceph/ceph/pull/60742 into this pr and add this line here?

> with-accounts: false